data-prepper-core/build.gradle

```diff
@@ -24,6 +24,7 @@ dependencies {
     implementation 'io.micrometer:micrometer-core'
     implementation 'io.micrometer:micrometer-registry-prometheus'
     implementation 'io.micrometer:micrometer-registry-cloudwatch2'
+    implementation 'javax.ws.rs:javax.ws.rs-api:2.1.1'
```

Comment:
I wouldn't add this dependency just for these constants. But, our `THIRD-PARTY` file does have this dependency, so I think it is fine.